### PR TITLE
Publish OSOR interview about OpenZaak

### PR DESCRIPTION
**Pull request by AlbaRoza**
Solves [this issue](https://github.com/open-zaak/open-zaak-website/issues/67)

Includes: 

- Text in Dutch
- Text in English

### CODE REVIEW

File: src/pages/news/2021-04-26-OSOR-interview.en.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+title: "OpenZaak featured in the European Open Source Observatory (OSOR)"
+lang: en
+template: default
+---
+
+# OpenZaak featured in the European Open Source Observatory (OSOR)
+
+The European portal [OSOR](https://joinup.ec.europa.eu/collection/open-source-observatory-osor), Open Source Observatory, published on April 7th an interview with Lazo Bozarov, Senior Supply Manager Digital Services at Utrecht Municipality about OpenZaak.
+
+In this text, Bozarov comments on the advantages that working in open source provides once the new [Dutch API (Application Programmable Interface) standard for case-oriented](https://www.vngrealisatie.nl/nieuws/api-standaarden-zaakgericht-werken-vanaf-1-april) has been implemented.
```
Comment (Ainali) [inline src/pages/news/2021-04-26-OSOR-interview.en.md]: Is a word missing here? Or does "case management" fit better than "case-oriented"?

File: src/pages/news/2021-04-26-OSOR-interview.en.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+You can read the interview [here](https://joinup.ec.europa.eu/collection/open-source-observatory-osor/news/we-liberated-our-own-data-silos-market).
```
Comment (Ainali) [inline src/pages/news/2021-04-26-OSOR-interview.en.md]:
```suggestion
[Read the interview on Joinup](https://joinup.ec.europa.eu/collection/open-source-observatory-osor/news/we-liberated-our-own-data-silos-market).
```

File: src/pages/news/2021-04-26-OSOR-interview.en.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+path: "/en/news/2021-04-26-OSOR-interview"
+date: "2021-04-26"
+title: "OpenZaak featured in the European Open Source Observatory (OSOR)"
+lang: en
+template: default
+---
+
+# OpenZaak featured in the European Open Source Observatory (OSOR)
+
+The European portal [OSOR](https://joinup.ec.europa.eu/collection/open-source-observatory-osor), Open Source Observatory, published on April 7th an interview with Lazo Bozarov, Senior Supply Manager Digital Services at Utrecht Municipality about OpenZaak.
```
Comment (Ainali) [inline src/pages/news/2021-04-26-OSOR-interview.en.md]:
```suggestion
The European portal [OSOR](https://joinup.ec.europa.eu/collection/open-source-observatory-osor), Open Source Observatory, published an interview with Lazo Bozarov, Senior Supply Manager Digital Services at Utrecht Municipality about OpenZaak on April 7th.
```

Comment (AlbaRoza) [reply src/pages/news/2021-04-26-OSOR-interview.en.md]: Does it really matter?

File: src/pages/news/2021-04-26-OSOR-interview.en.md (hunk 1 above)
Comment (AlbaRoza) [reply src/pages/news/2021-04-26-OSOR-interview.en.md]: I've changed it!

**Review by AlbaRoza — COMMENTED** [review]
second round!

File: src/pages/news/2021-04-26-OSOR-interview.en.md (hunk 3 above)
Comment (Ainali) [reply src/pages/news/2021-04-26-OSOR-interview.en.md]: Not semantically, but if we follow the journalistic method of leading with the most important message, I would say the date is the least important piece of information in this sentence.

File: src/pages/news/2021-04-26-OSOR-interview.en.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+In this text, Bozarov comments on the advantages that working in open source provides once the new [Dutch API (Application Programmable Interface) standard for case-management](https://www.vngrealisatie.nl/nieuws/api-standaarden-zaakgericht-werken-vanaf-1-april) has been implemented.
```
Comment (Ainali) [inline src/pages/news/2021-04-26-OSOR-interview.en.md]:
```suggestion
In this text, Bozarov comments on the advantages that working in open source provides once the new [Dutch API (Application Programmable Interface) standard for case management](https://www.vngrealisatie.nl/nieuws/api-standaarden-zaakgericht-werken-vanaf-1-april) has been implemented.
```

**Review by AlbaRoza — COMMENTED** [review]
Changes applied!

**Review by Ainali — APPROVED** [review]
English version looks good! (The Dutch look like Greek to me :) )

**Review by felixfaassen — APPROVED** [review]
Good to go! Dutch one is great!

**Pull request merged**